eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -271,7 +271,10 @@ jobs:
         fi
 
         if [[ -n "${{ parameters.crossRootFs }}" ]]; then
-          customEnvVars="$customEnvVars ROOTFS_DIR=${{ parameters.crossRootFs}} CROSSCOMPILE=1"
+          customEnvVars="$customEnvVars ROOTFS_DIR=${{ parameters.crossRootFs}}"
+          if [[ '${{ parameters.targetArchitecture }}' != 'wasm' ]]; then
+            customEnvVars="$customEnvVars CROSSCOMPILE=1"
+          fi
```

Comment:
which issues does this cause on wasm?

Reply:
There's some logic in gen-buildsys i think which sees CROSSCOMPILE=1 and does a build for $BuildOS-$TargetArch instead of $TargetOS-$TargetArch, something like that, anyway it fails on "linux-wasm isn't a thing"

Reply:
ok, would you mind filing an issue for this? I'd rather fix this in runtime than hardcoding this logic in the VMR (it's fine for now).